Python/Email_Scrape_From_Commits/install.sh

```diff
@@ -0,0 +1 @@
+pip install -r requirements.txt 
```

Comment:
I think that this script is unnecessary, why not simply use ```pip install -r requirements.txt```. In case you were going to trigger multiple commands, this would have made more sense to me.

Reply:
@vybhav72954  done 😄